ingest/lambdas/index-rectifier.js

```diff
@@ -0,0 +1,29 @@
+const ir = require('./index-rectifier');
+
+const handler = async () => {
+  const region = process.env.AWS_REGION;
```

Comment:
The usual, let's check and make sure these are set and valid.

Reply:
This env var should be set by Lambda: https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html but I can add a fallback value if you think it's necessary.